.github/workflows/ci.yml

```diff
@@ -0,0 +1,80 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '*'
+  pull_request:
```

Comment:
why is this dictionary entry empty? ôO

Reply:
Because you don't want to filter which pull request to select or not. All pull requests should run the CI, so we don't need anything in there.

Reply:
is it equivalent to the `- "*"` above?

Reply:
no, the `on` object should be a string (1 element), a list of strings (n elements) or a dict (n element with their configuration). If it's a dict, each value must itself be a dict (`configuration key: value`). The doc explicitely says to leave it empty if we don't need to configure anything:
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration

Reply:
ok.